### PR TITLE
ES: EP2 Ra file

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Story/Episode 2/ra_020040.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ra_020040_0050#0,"""Supongo que poder disfrutarme es un raro privilegio que me ha sido concedido por vivir tanto tiempo."""
+ra_020040_0060#0,"""Parece que estás haciendo bastante. He escuchado reportes de eventos aquí y allá."""
+ra_020040_0070#0,"""Puedes hacer lo que quieras, pero no te permito que interfieras. Eso sería demasiado aburrido."""
+ra_020040_0080#0,"""Después de vivir por casi una eternidad, a veces necesita algo de emoción. Usted está de acuerdo, ¿verdad?"""
+ra_020040_0090#0,"""Luther ya no es humano, pero no debes olvidar que él era uno."""
+ra_020040_0100#0,"""Se deja llevar. El temor a la falta de seriedad era su mayor temor como persona."""
+ra_020040_0110#0,"""Ésa podría ser su mayor fortaleza o mayor debilidad. Por lo menos, digno de mención."""
+ra_020040_0120#0,"""Eso es lo que hemos concluido. Con suerte, esa conclusión no es un error."""
+ra_020040_0130#0,"""Debes continuar como creas que necesitas. Esto es lo que todos desean."""
+text01#0,"""<br>Obtienes Matter board<br>"La luz que brilla en el borde de la esperanza.""""
```
Comment (alama) [inline Story/Episode 2/ra_020040.csv]: ra_020040.csv's text01#0 have an `"`, should be '\""`

**Comment by alama** [comment]
You still have aN unescape quote in the string

**Comment by NozomiRem** [comment]
#unescape#
¿no entiendo muy bien?
do not really understand?

File: Story/Episode 2/ra_025160.csv [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ra_025160_0070#0,"""2. No es muy diferente."""
+ra_025160_0080#0,"""Gracia, eh ... Si me comporto más modesta, será que me hace parecer más normal?"""
+ra_025160_0090#0,"""Sólo estás siendo amable conmigo. Puedes ser honesto."""
+ra_025160_0100#0,"""Sé muy bien que no soy normal, y nunca me he resentido de ese hecho."""
+ra_025160_0110#0,"""Oye, tal vez si actúo lo contrario ... Quiero decir, más modestamente, ¿me haría parecer más normal?"""
+ra_025160_0120#0,"""Ahem. ...Saludos, <%me>. ¿Cómo estás? Lo estoy haciendo bien. Espero con interés la conversación de hoy."""
+ra_025160_0130#0,"""...¿Por qué estás tan sorprendido? Fallé, eh. ¡Pero puedo actuar más elegante si lo intento!"""
+ra_025160_0140#0,"""De todos modos, voy a parar. Si empiezo a actuar de manera cálida y borrosa, siento que perderé la voluntad de pelear."""
+ra_025160_0150#0,"""Ese tipo de comportamiento me conviene mejor una vez que hayamos restaurado la paz a través de la galaxia."""
+ra_025160_0160#0,"""Hasta entonces, el acto de "chica normal" tendrá que esperar. ¡Lo guardaré para más tarde!"""
```
Comment (alama) [inline Story/Episode 2/ra_025160.csv]: More `\""` needed

File: Story/Episode 2/ra_020010.csv [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ra_020010_0220#0,"""Pero la razón podría ser que la primera persona en tocarlo fue él..."""
+ra_020010_0230#0,"""¡No me importa eso! ¡Sólo quiero destruir a la persona que estropeó nuestras vidas!"""
+ra_020010_0240#0,"""Quna, <%me> es la llave. ¿Lo entiendes?"""
+ra_020010_0250#0,"""Haré las cosas como yo decida. No tengo intención de trabajar contigo, Casra de los Tres Héroes."""
+ra_020010_0260#0,"""Lo sabes tú mismo, ¿no?"""
+ra_020010_0270#0,"""Sí, lo haces Quna. Si todo termina, este cuello, esta vida puedes hacer lo que quieras con ella."""
+ra_020010_0280#0,"""Ese es nuestro contrato. Ese es el único riesgo que acordaste mantener."""
+ra_020010_0290#0,"""Realmente odio tu cara. ¿Qué estás buscando?"""
+ra_020010_0300#0,"""A partir de este punto en el tiempo, lo mismo que usted."""
+ra_020010_0310#0,"""Hm."""
```
Comment (alama) [inline Story/Episode 2/ra_020010.csv]: ra_020010_0310#0 is in EN

File: Story/Episode 2/ra_020440.csv [hunk 4]
```diff
@@ -0,0 +1,23 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+ra_020440_0001#0,"""A.P.238/6/20"""
+ra_020440_0010#0,"""Oh, <%me>, bienvenido de nuevo. Siempre parece estar trabajando duro."""
+ra_020440_0020#0,"""He oído que una nueva área fue descubierta en el planeta Vopar."""
+ra_020440_0030#0,"""¿Cómo es? Siempre me hace feliz escuchar sus experiencias."""
+ra_020440_0040#0,"""¿Una instalación misteriosa en el fondo del océano? Me pregunto para qué era?"""
+ra_020440_0050#0,"""Me pregunto si fue por excavar recursos como la mina de Lilipa."""
+ra_020440_0090#0,"""Hm...."""
```
Comment (alama) [inline Story/Episode 2/ra_020440.csv]: ra_020440_0090#0 still in EN

File: Story/Episode 2/ra_020500.csv [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+ra_020500_0040#0,"""Pero me temo que es un no. Aquí las hermanas tenemos cosas que podemos y no podemos hacer."""
+ra_020500_0050#0,"""El incomprensible balbuceo de Patty a un lado... Eso es lo mismo que nos preguntó Theodore."""
+ra_020500_0060#0,"""Pero todo lo que tengo es disculpas. Como dijo Patty, no podemos ayudarte con eso.."""
+ra_020500_0070#0,"""A pesar de que no se supone que es una información tan importante, seguimos corriendo peligro..."""
+ra_020500_0080#0,"""El informe de la víctima para ese evento ha sido completamente clasificado. Nadie puede verlo..."""
+ra_020500_0090#0,"""No podemos encontrar nada usando nuestros métodos habituales, tampoco. ¿Porque eso? Simplemente aumenta el misterio..."""
+ra_020500_0100#0,"""Bueno, de alguna manera conseguimos un informe sobre los difuntos, pero no parece que eso sea lo que estás buscando..."""
+ra_020500_0110#0,"""Pero, no se preocupe! ¡Si no podemos sacarlo electrónicamente, vamos a ir de boca en boca!"""
+ra_020500_0120#0,"""...Bueno, eso suele ser el caso, pero ... ¿tienes pistas, Patty?"""
+ra_020500_0130#0,"""Nope!"""
```
Comment (alama) [inline Story/Episode 2/ra_020500.csv]: ra_020500_0130#0 still in EN

File: Story/Episode 2/ra_025100.csv [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+ra_025100_0030#0,"""Parece que independientemente de cuánto tiempo atrás viajas, solo puedes permanecer un rato. Terminemos lo que necesitamos--"""
+ra_025100_0040#0,"""...Ya parece que esto no va a ser fácil. Recuerde, no puede pedir ayuda en esta línea de tiempo. ¡¡Quédate alerta!!"""
+ra_025100_0050#0,"""¿¡Fuego amigo!? ¿Quién sería tan imprudente?"""
+ra_025100_0060#0,"""Huh? ¿Creías que dijiste que sólo había Darkers en esta vecindad?"""
+ra_025100_0070#0,"""...oh, Ya veo. Una forma humanoide. Debes ser un Dark Falz, ¿verdad?"""
+ra_025100_0080#0,"""...Error. Los parámetros no coinciden con los valores observados. Claris Claes, Este individuo no es un Darker."""
+ra_025100_0090#0,"""...eh? EEH!? Pe-pero usted dijo ... así que yo...!!"""
+ra_025100_0100#0,"""No hice ninguna declaración semejante. Sólo hablaba de tal posibilidad."""
+ra_025100_0103#0,"""Dios mío! Tú...ngh!"""
+ra_025100_0106#0,"""A-aha-haha...um..."""
```
Comment (alama) [inline Story/Episode 2/ra_025100.csv]: ra_025100_0106#0 still in EN

File: Story/Episode 2/ra_025150.csv [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+name02#0,"""？？？"""
+ra_025150_0001#0,"""A.P.228/8/1"""
+ra_025150_0010#0,"""...Ugh, ¿Por qué dije esas palabras?"""
+ra_025150_0020#0,"""Primero la invito a la conversación, y luego la rechazo. ¿Qué estaba pensando?"""
+ra_025150_0030#0,"""Soy tan hipócrita..."""
+ra_025150_0040#0,"""Estoy segura de que me odian. ¡Nunca nos encontraremos otra vez! Waaaaah--!"""
+ra_025150_0050#0,"""¿¡Solo es natural!? Incluso se estremece de vez en cuando !?"""
+ra_025150_0060#0,"""¡Cállate! Por lo menos podrías animarme! ¡Es culpa tuya!"""
+ra_025150_0070#0,"""......suspiro."""
+ra_025150_0080#0,"""...huh?"""
```
Comment (alama) [inline Story/Episode 2/ra_025150.csv]: ra_025150_0080#0 still in EN?

File: Story/Episode 2/ra_020010.csv (hunk 3 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_020010.csv]: Seems it detect some lines with only an onomatopoeia like untranslated.

File: Story/Episode 2/ra_020440.csv (hunk 4 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_020440.csv]: Seems it detect some lines with only an onomatopoeia like untranslated.

File: Story/Episode 2/ra_020500.csv (hunk 5 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_020500.csv]: Well, here "Nope!" can be translated as "No!"

File: Story/Episode 2/ra_025100.csv (hunk 6 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_025100.csv]: Seems it detect some lines with only an onomatopoeia like untranslated.

File: Story/Episode 2/ra_025150.csv (hunk 7 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_025150.csv]: Seems it detect some lines with only an onomatopoeia like untranslated.

File: Story/Episode 2/ra_020440.csv (hunk 4 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_020440.csv]: in that case ""Hm....""

File: Story/Episode 2/ra_020010.csv (hunk 3 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_020010.csv]: in that case ""Hm....""

File: Story/Episode 2/ra_025100.csv (hunk 6 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_025100.csv]: On that """A-aha-haha...um..."""

File: Story/Episode 2/ra_025150.csv (hunk 7 above)
Comment (TheBlazter) [reply Story/Episode 2/ra_025150.csv]: in that case ""...huh?"""

**Pull request merged**